libraries/Preferences/src/Preferences.cpp

```diff
@@ -53,10 +53,14 @@ bool Preferences::begin(const char * name, bool readOnly, const char* partition_
     return true;
 }
 
-void Preferences::end(){
+void Preferences::end(){                                         // modified to add an nvs_commit()
     if(!_started){
         return;
     }
+    esp_err_t err = nvs_commit(_handle);                         // to undo changes: delete the lines from here...
```

Comment:
Since every other method which does some modifications already calls nvs_commit, is this call actually necessary?